public/app/features/dashboard/state/DashboardModel.ts

```diff
@@ -365,11 +370,30 @@ export class DashboardModel {
   exitViewPanel(panel: PanelModel) {
     this.panelInView = undefined;
     panel.setIsViewing(false);
+    this.refreshIfChangeAffectsAllPanels();
   }
 
   exitPanelEditor() {
     this.panelInEdit!.destroy();
     this.panelInEdit = undefined;
+    this.refreshIfChangeAffectsAllPanels();
+  }
+
+  changeAffectsAllPanels() {
```

Comment:
this naming implies a boolean return value for me (ie reads like a question) not a state change action.

Reply:
Yeah, naming I'll try to figure something out.

Reply:
Keeping this for now.

Reply:
Fixed